terraform/ec2/amis.tf

```diff
@@ -221,6 +267,12 @@ EOF
       family = "linux"
       arch = "arm64"
       instance_type = "t4g.nano"
+      user_data = <<EOF
```

Comment:
I think all AMI owned by amz has ssm by default? Though it won't hurt to reinstall ssm even if AMI has ssm agent as it simply update ssm agent.